peachjam/adapters/adapters.py

```diff
@@ -106,7 +106,12 @@ def update_document(self, url):
 
         logger.info(f"Updating document ... {url}")
 
-        document = self.client_get(f"{url}.json").json()
+        try:
+            document = self.client_get(f"{url}.json").json()
+        except requests.HTTPError as e:
+            logger.error(f"An error has occurred: {e}")
+            return
+
```

Comment:
You can try it like this:
```suggestion
            if e.response.status_code == 404:
                return
            else:
                raise e          

```